packages/console/src/pages/SignInExperience/tabs/Others/components/ManageLanguage/LanguageEditor/LanguageDetails.tsx

```diff
@@ -45,7 +45,11 @@ const LanguageDetails = () => {
   const fetcher = useSwrFetcher<CustomPhraseResponse>(fetchApi);
 
   const translationEntries = useMemo(
-    () => Object.entries((isBuiltIn ? resource[selectedLanguage] : en).translation),
+    () =>
+      Object.entries((isBuiltIn ? resource[selectedLanguage] : en).translation).filter(
+        // Filter out the demo app phrases in AC
+        ([groupKey]) => groupKey !== 'demo_app'
```

Comment:
should this be hidden by api instead?

Reply:
i'm ok with hide in frontend, but this looks a bit weird

Reply:
I had the same thoughts. But it looks like the field keys are directly imported from the phrases-ui package instead of calling an API.  cc: @xiaoyijun for any better ideas. One thing cross my mind is reorg the phrases-ui 's key structure, and extract the demo-app phrases from 'translation' namespaces.

Reply:
Due to the shared use of the same LocalePhrase data between the front-end and back-end, it is recommended to follow Simeng's suggestion and separate the demo from the LocalePhrase.

Reply:
namespace is good let's go with it. not a blocker can do later

Reply:
Cool issue added to the eng quality Q1